yolo/yamlCreation.py

```diff
@@ -0,0 +1,68 @@
+import platform
+import glob
+import os
+import yaml
+
+def gatherFolderPath(queriedFolderName: str) -> str:
```

Comment:
seems to work as expected - at least on unix I can confirm that - and looks well so far. 
I would likely merge thi function into another module at a later point, but thats not the scope this branch!